File: CS/AutoScaling/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraPrinting;
using DevExpress.XtraReports.UI;
using DevExpress.XtraPrinting.Preview;
// ...

namespace AutoScaling {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            ReportPrintTool printTool = new ReportPrintTool(new XtraReport1());
            printTool.Report.CreateDocument(false);
            printTool.PreviewForm.Load += new EventHandler(PreviewForm_Load);
            printTool.ShowPreviewDialog();
        }

        void PreviewForm_Load(object sender, EventArgs e) {
            PrintPreviewFormEx frm = (PrintPreviewFormEx)sender;
            frm.PrintingSystem.ExecCommand(PrintingSystemCommand.Scale, new object[] { 0.7f });
        }

    }
}

File: VB/AutoScaling/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraReports.UI
Imports DevExpress.XtraPrinting.Preview
' ...

Namespace AutoScaling
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim printTool As New ReportPrintTool(New XtraReport1())
			printTool.Report.CreateDocument(False)
			AddHandler printTool.PreviewForm.Load, AddressOf PreviewForm_Load
			printTool.ShowPreviewDialog()
		End Sub

		Private Sub PreviewForm_Load(ByVal sender As Object, ByVal e As EventArgs)
			Dim frm As PrintPreviewFormEx = CType(sender, PrintPreviewFormEx)
			frm.PrintingSystem.ExecCommand(PrintingSystemCommand.Scale, New Object() { 0.7f })
		End Sub

	End Class
End Namespace

